AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/src/Mutations.dfy

```diff
@@ -17,12 +17,17 @@ module {:options "/functionSyntax:4" } Mutations {
   import KmsArn
   import KMSKeystoreOperations
   import DefaultKeyStorageInterface
+  import GetKeys
 
   import Types = AwsCryptographyKeyStoreAdminTypes
   import StateStrucs = MutationStateStructures
   import MutationErrorRefinement
   import KmsUtils
 
+  datatype ActiveVerificationHolder =
```

Comment:
add a comment on why you created this new datatype and what it's used for

Reply:
Done at [020c0b3](https://github.com/aws/aws-cryptographic-material-providers-library/pull/1442/commits/020c0b308c2cb7382c8db461d415db40be7fe63a)